ADVANCED.md

```diff
@@ -1110,6 +1116,8 @@ return (
 | `searchQuery`               | [Query Config](#query-config)                                           | optional  | {}      | Configuration options for the main search query.                                                                                                                                                                             |
 | `trackUrlState`             | Boolean                                                                 | optional  | true    | By default, [Request State](#request-state) will be synced with the browser url. To turn this off, pass `false`.                                                                                                             |
 | `urlPushDebounceLength`     | Integer                                                                 | optional  | 500     | The amount of time in milliseconds to debounce/delay updating the browser url after the UI update. This, for example, prevents excessive history entries while a user is still typing in a live search box.                  |
+| `a11yNotifications`         | Boolean                                                                 | optional  | false   | Search UI will create a screen reader accessible live region to announce search results & other actions. This will be turned on by default in our 2.0 release.                                                               |
```

Comment:
I like this copyedit! I think my only thought would be changing `invisible` to `visually hidden`, since that has a distinct meaning/context for folks working in accessibility.

So end result:

```
Search UI will create an visually hidden live region to announce search results & other actions to screen reader users.
```

I also tweaked the last clause a bit to make it a bit more succinct - thoughts?

Re: 2.0 release notes - I think it's important to inform devs of upcoming breaking changes / deprecations like this, and there's no such thing as over-information when it comes to this kind of thing (better devs be mildly annoyed with repeated warnings than to be taken by surprise when their app breaks). Let me know if that makes sense to you!

Reply:
Awesome, I totally agree with both the text changes and the 2.0 thoughts!